packages/cli/src/templates/project/package-json.ts

```diff
@@ -37,7 +39,7 @@ export const template = `{
   "scripts": {
     "lint:check": "eslint --ext '.js,.ts' **/*.ts",
     "lint:fix": "eslint --quiet --fix --ext '.js,.ts' **/*.ts",
-    "compile": "npx tsc -b tsconfig.json",
+    "compile": "npx ttypescript -b tsconfig.json",
```

Comment:
Yes, but when using npx with `ttsc` it fails on retrieving the package. That's why I'm requesting the whole package instead of using just the binary

Reply:
🤔  Oh, that's when `ttypescript` is not installed (or it is not listed in the dependencies), so npx tries to find that package and install it. This is not ideal because it installs a "random version" (normally the latest).
I guess we should see why it is failing (it is not failing in the integration-tests package and we are using `ttsc` there (if I recall correctly)

Reply:
Let's try to sort this, to have it ready for merge 💪🏻

Reply:
After a conversation with @alvaroloes, we found out that it's necessary to use `npm >= 7.0.0` in order to have `npx` behaving properly.

Reply:
I'll update the docs

Reply:
Also, turns out that `npx`it's not necessary when using npm scripts on the `package.json` file as npm is able to retrieve the binary from the root dependencies. You can see a deeper explanation here: https://www.freecodecamp.org/news/npm-vs-npx-whats-the-difference/#npm-the-package-manager